posthog/api/feature_flag.py

```diff
@@ -0,0 +1,31 @@
+from posthog.models import FeatureFlag
+from posthog.api.user import UserSerializer
+from rest_framework import request, serializers, viewsets
+from django.db.models import QuerySet
+from typing import List, Dict, Any
+import json
+
+class FeatureFlagSerializer(serializers.HyperlinkedModelSerializer):
+    created_by = UserSerializer(required=False, read_only=True)
+
+    class Meta:
+        model = FeatureFlag
+        fields = ['id', 'name', 'key', 'rollout_percentage', 'filters', 'deleted', 'active', 'created_by', 'created_at']
```

Comment:
Would be worth it to add a `expires_at` date so people could have a predetermined end date for the feature flag, it would even be possible to have some web hook trigger when it's created, updated, deleted or expiring :)

Reply:
Yep I like that idea and would love to see a ticket for it, but I'd suggest it's out of scope for this specific PR!

Reply:
That's true best to get a basic version of it out and then expand on it :)